vedant

vedantpotekar opened the pull request:
I've added a file. Please add it if you find it useful.

vedantpotekar committed:
Vedant has added a file.

vedantpotekar committed:
a file has been added by vedant

prathamateasycode reviewed (changes requested):
Change file name,
File name should be: (Problem ID).py in this format

vedantpotekar committed:
made the required changes

vedantpotekar commented:
I've made the needful changes. The earlier file was a text document, now I've renamed and changed it.
Further, I've added the solution python file with the required file name.
Thank you.

prathamateasycode reviewed (approved):
These changes are acceptable.
But please add C solution as well
